helpers

```diff
@@ -115,3 +115,15 @@ get_ceph_volume_lvm_list ()
     fi
 }
 export -f get_ceph_volume_lvm_list
+
+get_lvm2_lvs ()
+{
+    local sos_path="${DATA_ROOT}/sos_commands/lvm2/lvs_-a_-o_lv_tags_devices*"
+    if [ -e $sos_path ]; then
+        cat $sos_path
+    elif ! [ -d "${DATA_ROOT}sos_commands" ] && which lvs >/dev/null; then
+        lvs -a -o lv_tags,devices,lv_kernel_read_ahead,lv_read_ahead,stripes,stripesize
```

Comment:
is this definitely going to produce equivalent output to the sosreport version?

Reply:
This is based on what's used in sos: https://github.com/sosreport/sos/blob/master/sos/report/plugins/lvm2.py#L68
I have previously tested this in sts bastion and that awk line (from 01ceph) retrieved the same info correctly.

Reply:
I'll test again in sts bastion, just to be sure.

Reply:
ok, I probably didn't check sts but few sosreports. Looks like all these options aren't needed - just the 2 are sufficient.